.travis.yml

```diff
@@ -39,7 +39,13 @@ script:
   - |
     if [[ ! -z "$WP_VERSION" ]] ; then
       phpunit
+    fi
+  - |
+    if [[ ! -z "$WP_VERSION" ]] ; then
```

Comment:
Is there is specific reason why the phpunit needs WP_VERSION, when it is not needed in the starter theme? - https://github.com/alleyinteractive/wp-starter-theme/blob/master/.travis.yml#L126-L127